ooasp/encodings/viz_config_nodes.lp

```diff
@@ -0,0 +1,35 @@
+% domain_node(Class,Object_id) :- ooasp_domain(Class,Object_id), not ooasp_isa(_,Object_id).
+% isa_node(ClassIsA,Object_id) :- ooasp_domain(Class,Object_id), ooasp_isa(ClassIsA,Object_id), not ooasp_isa_leaf(_,Object_id).
+% leaf_node(Class,Object_id) :- ooasp_isa_leaf(Class,Object_id), ooasp_class(Class).
+% isa_smallest(SUPER,Object_id) :-
+%     ooasp_isa(SUPER,Object_id),
+% 	not ooasp_isa(SUB, Object_id): ooasp_subclass(SUB,SUPER).
+% ooasp_isa_leaf(C,Object_id) :-
+%     ooasp_isa(C,Object_id),
+% 	ooasp_leafclass(C).
+
+% any_node(Object_id):- domain_node(Class,Object_id).
+% any_node(Object_id):- leaf_node(Class,Object_id).
+% any_node(Object_id):- isa_node(Class,Object_id).
+
+
+% unused_node(Class,Object_id) :- ooasp_domain(Class,Object_id), not _all(ooasp_isa(_,Object_id)).
+% unused_node(Class,Object_id) :- ooasp_domain(Class,Object_id), _clinguin_browsing, not ooasp_isa(_,Object_id).
+
+% infered_class_node(Class,Object_id) :- ooasp_domain(Class,Object_id), not _all(ooasp_isa(_,Object_id)).
+% unused_node(Class,Object_id) :- ooasp_domain(Class,Object_id), _clinguin_browsing, not ooasp_isa(_,Object_id).
+
```

Comment:
Why is this commented-out?

Reply:
Must be another leftover from a previous version.
I will remove it.